### ISSUE TITLE
Feature request: zellij attach --create

### ISSUE THREAD

**Issue by srid** [zellij-org/zellij#541]
A command that automatically creates (if it doesn't exist) and joins the given named-session would be useful.

It can come in handy in creating quick project-specific sessions, for instance:

```
zellij attach --create $(basename `pwd`)
```

**Comment by srid** [comment]
Tmux analogue

```
tmux new-session -A -s Foo
```

**Comment by a-kenji** [comment]
Thank you for taking the time creating this issue!
This would be a welcome addition.

**Comment by pedrohjordao** [comment]
It seems like it should be easy enough. I would like to take a stab at it.

Any traps I should be aware of?

**Comment by imsnif** [comment]
@pedrohjordao - tbh I'm personally not familiar with this part of the code very well, the person who wrote it is a little busy in the coming weeks (which is why I don't want to tag him :) ). Which is to say that I'm not aware of any specific traps - but I'm sure we can figure this out. Feel free to reach out here or on the Discord #development channel if you have any questions or would like some guidance.

**Comment by kunalmohan** [comment]
Hi @pedrohjordao! I don't think there are any traps in this part of the code. The logic for implementing this will go [here](https://github.com/zellij-org/zellij/blob/7f67e6d97f158a00d1f2c21326fe320388ec8446/src/main.rs#L57-L76). This block of code can be a bit tricky to understand, so if you need any help, feel free to ping me (here or on discord) 🙂

**Comment by kunalmohan** [comment]
Implemented in #731
